### PR TITLE
update compose to 1.6.2

### PR DESCRIPTION
**Pull request by 2k0ri**
To support version 2 syntax of `docker-compose.yml`.

### CODE REVIEW

File: op-enable.go [hunk 1]
```diff
@@ -250,6 +250,10 @@ func composeUp(d driver.DistroDriver, json map[string]interface{}, env map[strin
 	defer os.Unsetenv("COMPOSE_HTTP_TIMEOUT")
 	defer os.Unsetenv("DOCKER_CLIENT_TIMEOUT")
 
+	// set docker-compose.yml path to share directory with docker-compose container.
+	os.Setenv("COMPOSE_FILE", ymlPath)
```
Comment (ahmetalpbalkan) [inline op-enable.go]: Thanks for the PR, I'm not aware how is this different than passing compose file with `-f ymlPath`.

**Comment by 2k0ri** [comment]
In [docker-compose' run.sh](https://github.com/docker/compose/blob/master/script/run/run.sh#L37-L39), the file should be provided through `COMPOSE_FILE` to be accessible from docker-compose container's namespace(I found TODO "also check --file argument" in it).

**Comment by ahmetalpbalkan** [comment]
@2k0ri I'm not sure what “docker-compose container's namespace” means. We are not running compose inside a container and [from the docs](https://docs.docker.com/compose/reference/envvars/) I see `COMPOSE_FILE` is the same thing as `-f`, which we already have. So if you can remove the os.Setenv/Unsetenv, I can merge.

**Comment by aucfan-kori** [comment]
I was misunderstanding its installation processes. I reverted the commit.

**Comment by ahmetalpbalkan** [comment]
@2k0ri this is now done. Current version of docker-extension as of today uses compose-1.6.2.

**Pull request closed without merging**